spec/passaporte_web/notification_spec.rb

```diff
@@ -139,7 +139,7 @@
         end
         it "should create the Notification on PassaporteWeb, authenticated as the application" do
           notification.should_not be_persisted
-          notification.save('application').should be_true
+          expect(notification.save('application')).to be_truthy
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.